app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  has_many :pets
+  has_many :favorites
+  has_many :applications
+
+  validates_presence_of :username
```

Comment:
Are we including User name? iu know we discussed this, but can't recall where we landed. i prefer to have both user name and email, but want to make sure everyone is on the same page.